spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -131,365 +131,10 @@
-              click_on t('forms.buttons.upload_photos')
-              expect(page).to have_current_path(idv_document_capture_url)
-              expect_step_indicator_current_step(t('step_indicator.flows.idv.verify_id'))
-              expect_doc_capture_page_header(t('doc_auth.headings.document_capture_with_selfie'))
-              expect_doc_capture_id_subheader
-              expect_doc_capture_selfie_subheader
-              attach_liveness_images
-              submit_images
-
-              expect(page).to have_current_path(idv_ssn_url)
-              expect_costing_for_document
-              expect(DocAuthLog.find_by(user_id: @user.id).state).to eq('MT')
-
-              expect(page).to have_current_path(idv_ssn_url)
-              fill_out_ssn_form_ok
-              click_idv_continue
-              complete_verify_step
-              expect(page).to have_current_path(idv_phone_url)
-            end
-          end
-
-          context 'when ipp is enabled' do
-            let(:in_person_doc_auth_button_enabled) { true }
-            let(:sp_ipp_enabled) { true }
-
-            before do
-              allow(IdentityConfig.store).to receive(:in_person_doc_auth_button_enabled).
-                and_return(in_person_doc_auth_button_enabled)
-              allow(Idv::InPersonConfig).to receive(:enabled_for_issuer?).with(anything).
-                and_return(sp_ipp_enabled)
-            end
-
-            describe 'when ipp is selected' do
-              it 'proceed to the next page and start ipp' do
-                perform_in_browser(:desktop) do
-                  visit_idp_from_oidc_sp_with_ial2(facial_match_required: true)
-                  sign_in_and_2fa_user(@user)
-                  complete_doc_auth_steps_before_hybrid_handoff_step
-                  # we still have option to continue on handoff, since it's desktop no skip_hand_off
-                  expect(page).to have_current_path(idv_hybrid_handoff_path)
-                  expect(page).to have_content(t('doc_auth.headings.hybrid_handoff_selfie'))
-                  click_on t('in_person_proofing.headings.prepare')
-                  expect(page).to have_current_path(
-                    idv_document_capture_path({ step: 'hybrid_handoff' }),
-                  )
-                  expect_step_indicator_current_step(
-                    t('step_indicator.flows.idv.find_a_post_office'),
-                  )
-                  expect_doc_capture_page_header(t('in_person_proofing.headings.prepare'))
-                end
-              end
-            end
-          end
-        end
-      end
-    end
-  end
-
   context 'split doc auth flow', allow_browser_log: true do
     before do
       allow(IdentityConfig.store).to receive(:use_vot_in_sp_requests).and_return(true)
       allow(IdentityConfig.store).to receive(:doc_auth_selfie_capture_enabled).and_return(true)
-      allow(IdentityConfig.store).to receive(:doc_auth_separate_pages_enabled).and_return(true)
```

Comment:
it appears that doc_auth_selfie_capture_enabled no longer exists 

this would be a great opportunity to remove the line above:
`allow(IdentityConfig.store).to receive(:doc_auth_selfie_capture_enabled).and_return(true)`